### ISSUE TITLE
Find a way to make template use default file by default 

### ISSUE THREAD

**Issue by inkhey** [tracim/tracim#2204]
## Bug description & symptom

Currently, we need to have template path in config file required because we do need absolute path to file. We should find a way to use default template file without requiring configuration. This will be useful for migration when new needed feature required a new template.

from : 

```
I see these configs are required: they are added on all configs. Maybe these configs can be optional + default value ?
Related issue (dot not change anything yet): #2018

```
_Originally posted by @buxx in https://github.com/tracim/tracim/pull/2195_

## Expected

- fix issue with absolute path to make template allowing default value without config file.

## Implemented solution

see https://github.com/tracim/tracim/pull/2575
- make template use default value by default
- default value for almost all parameters
- update "tracimcli dev parameters" command to allow more thing like showing default values of parameters.

**Comment by inkhey** [comment]
:warning:  we should verify if fix #2575 fix #2018 too.

**Comment by PhilippeAccorsi** [comment]
Tested and verified on branch. (wait merge to close)
